datastore/elastic_search/query.py

```diff
@@ -98,6 +99,57 @@ def ngrams_query(connection, index_name, doc_type, entity_name, ngrams_list, fuz
     return ngram_results
 
 
+def user_says_query(connection, index_name, doc_type, entity_name, user_typed_sentence, fuzziness_threshold,
```

Comment:
Remove 'user_says' from `user_says_query` and 'user_typed' from `user_typed_sentence`.

Refactor to something else like `full_text_query` or `simple_text_query`. Anything meaningful without 'user' goes